src/main/java/lottoauto/domain/LottoService.java

```diff
@@ -2,17 +2,13 @@
 
 import java.util.ArrayList;
 import java.util.List;
-import lottoauto.model.Constant;
 import lottoauto.model.Lotto;
 import lottoauto.model.Lottos;
 
 public class LottoService {
 
-    public Lottos generateLottoNumber(int amount) {
-        int quantity = getQuantity(amount);
-        amountValidation(amount);
-        List<Lotto> lotto = generateLotto(quantity);
-        return new Lottos(lotto);
+    public Lottos generateLottoNumber(int quantity) {
```

Comment:
생성한 LottoRequest를 받아서 처리할 수 있을 것 같습니다
```suggestion
    public Lottos generateLottoNumber(LottoRequest request) {
```